README.md

```diff
@@ -76,6 +76,23 @@ To stop the docker environment, run:
   docker-compose down
   ```
 
+## Running custom shell scripts on boot
+
+This container supports running arbitrary shell scripts on container boot. Any `*.sh` file located under `/docker-entrypoint.d` directory will be executed inside the container just before starting stackstorm services.
+
+For example, if you want to modify `/etc/st2/st2.conf` to set `system_packs_base_path` parameter, create `modify-st2-config.sh` with the follwing content:
+
+```
+/bin/bash
+crudini --set /etc/st2/st2.conf content system_packs_base_path /opt/stackstorm/custom_packs
+```
+
+Then bind mount it to `/docker-entrypoint.d/modify-st2-config.sh`
```

Comment:
Is it worth adding an example of how to bind mount, a) with `docker run` on command line, and b) using `docker-compose.yml`? Anyone who google's `bind mount docker` will see pretty quick how it's done, but it could save some trouble if they don't already know.